build/integration/features/bootstrap/WebDav.php

```diff
@@ -1112,4 +1124,32 @@ public function theUploadShouldFailOnObjectStorage() {
 			$this->theHTTPStatusCodeShouldBe(500);
 		}
 	}
+
+	/**
+	 * @return array
+	 * @throws Exception
+	 */
+	private function convertResponseToDavSingleEntry(): array {
+		$results = $this->convertResponseToDavEntries();
+		if (count($results) <> 1) {
```

Comment:
To match the exception case, otherwise I'd prefer to use `!==` 
```suggestion
		if (count($results) > 1) {
```

Reply:
the next array_shift will fail :(

Reply:
I changed the exception message, thanks for the review !